libraries/api/src/index.js

```diff
@@ -32,6 +36,53 @@ const ping = {
   },
 };
 
+/**
+ * A load balancer heartbeat method, added automatically to every service
+ * Following Dockerflow standards https://github.com/mozilla-services/Dockerflow/#containerized-app-requirements
+ * Can likely remove the /ping endpoint but I left for backwards compatibility
+ */
+const lbHeartbeat = {
+  method: 'get',
+  route: '/__lbheartbeat__',
+  name: 'lbheartbeat',
+  stability: 'stable',
+  title: 'Load Balancer Heartbeat',
+  category: 'Load Balancer Heartbeat',
+  description: [
+    'Respond without doing anything.',
+    'This endpoint is used to check that the service is up.',
+  ].join('\n'),
+  handler: function(_req, res) {
+    res.status(200).json({});
+  },
+};
+
+/**
+ * A version method, added automatically to every service
+ * Following Dockerflow standards https://github.com/mozilla-services/Dockerflow/#containerized-app-requirements
+ */
+const version = {
+  method: 'get',
+  route: '/__version__',
+  name: 'version',
+  stability: 'stable',
+  title: 'Taskcluster Version',
+  category: 'Taskcluster Version',
+  description: [
+    'Respond with the JSON version object.',
+    'https://github.com/mozilla-services/Dockerflow/blob/main/docs/version_object.md',
+  ].join('\n'),
+  handler: function(_req, res) {
+    try {
+      const taskclusterVersionFile = path.resolve(REPO_ROOT, 'version.json');
+      const taskclusterVersion = fs.readFileSync(taskclusterVersionFile).toString().trim();
```

Comment:
this should be done outside of the handler. `version.json` will not be changed during whole deployment, so there is no need to do a blocking read inside the handler, and pause event loop for other requests.

Reply:
or at least, if this is dynamic (which is probably not) we can do an async call here to avoid blocking event loop

Reply:
Good call, thanks @lotas